AtReconstruction/AtFilter/AtSCACorrect.h

```diff
@@ -23,24 +23,34 @@ class AtSCACorrect : public AtFilter {
    using RawEventPtr = std::unique_ptr<AtRawEvent>;
    AtMapPtr fMap;
 
-   RawEventPtr fBaseline;
-   RawEventPtr fPhase;
+   RawEventPtr fRawEvent;
+
+   Bool_t fDoBaseline;
+   Bool_t fDoPhase;
+   Bool_t fUseChanZero;
+
+   TString fBaseAugName;
+   TString fPhaseAugName;
 
 public:
-   AtSCACorrect(AtMapPtr map, TString baselineFilename, TString baselineEventName, TString phaseFilename,
-                TString phaseEventName);
-   AtSCACorrect(AtMapPtr map, RawEventPtr baseline, RawEventPtr phase);
+   AtSCACorrect(AtMapPtr map, TString filename, TString eventName, TString baselineAug, TString phaseAug);
+   AtSCACorrect(AtMapPtr map, RawEventPtr rawEvent, TString baselineAug, TString phaseAug);
 
    virtual void Init() override {}
    virtual void InitEvent(AtRawEvent *) override {}
    virtual bool IsGoodEvent() override { return true; }
 
    virtual void Filter(AtPad *pad, AtPadReference *padReference) override;
 
+   void DoBaseline(Bool_t val) { fDoBaseline = val; }
+   void DoPhase(Bool_t val) { fDoPhase = val; }
+
+   void UseChannelZero() { fUseChanZero = true; }
```

Comment:
These need descriptions as doxygen comments. I'm guessing DoBaseline uses the baseline correction in an augment somewhere rather than an AtRawEvent like before to do a flat baseline subtraction. And DoPhase removes the phase effect stored in an augment rather than an AtRawEvent like before. But I'm not sure by the name alone what UseChannelZero does.